modules/jaeger-removing-instance.adoc

```diff
@@ -0,0 +1,27 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-removing.adoc
+////
+
+[id="jaeger-removing_{context}"]
+= Removing a Jaeger instance using the web console
+
+====
+NOTE: When deleting an instance that uses the in-memory storage, all data will be permanently lost. The Jaeger Operator will not remove data from permanent storage, for example, Elasticsearch.
```

Comment:
Not sure "Jaeger Operator" is relevant in the second sentence - Maybe something like "Data stored in a persistent storage (such as Elasticsearch) will not be deleted when a Jaeger instance is removed."?